internal/ingress/controller/store/store_test.go

```diff
@@ -46,6 +46,7 @@ import (
 )
 
 func TestStore(t *testing.T) {
+	t.Skip("test is skipped because rancher doesn't build it in kubernetes")
```

Comment:
can we take a quick peek at skipping these tests via the ./scripts/test file instead of changing the code?

Reply:
I've removed these `t.Skip`s